Python Tutorial Exercises(Week 5)/#4.py

```diff
@@ -0,0 +1,26 @@
+'''
+use if len(i) >= 8 to find all the strings that have at least 8 characters or more.
```

Comment:
What does this overall script do?

Reply:
Do you want me to remove those code lines? I think it help the person who reads my code to have a better understanding about my code.

Reply:
I just deleted all the code lines from the document.

Reply:
It's all good to have those code lines, but it just needed a high level description for the script eg "This uses list comprehension to validate a list of passwords based on the below criteria"